src/api-v1.js

```diff
@@ -665,13 +666,17 @@ api.declare({
       token: token,
     });
 
+    let workerId = slugid.v4();
```

Comment:
Is there any length limit on the worker ID from our standpoint since it'll be included in scopes?